### PR TITLE
[BPK-1936] Add decision for previews

### PR DESCRIPTION
**Pull request by tiagoengel**

### CODE REVIEW

File: decisions/components-preview.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Component Preview (a.k.a stories)
+
+## Decision
+- All previews should respond to a shortcut to hide the navbar (CMD+D).
+- The component being previewed should fit in the screen in portrait mode *without* the navbar.
+- The IOS preview should be as close as possible as the Android version (Always respecting any platform specific behaviour).
+- Diferent configurations (e.g. normal text and emphasized text) should have different previews. In this case clicking the main story should show a new list view with all configs.
+- When converting a component from react native the previews should be as close as possible to the react native version.
```
Comment (georgegillams) [inline decisions/components-preview.md]: I would possibly remove this line - we made the mistake of making our RN stuff too "webby". If anything we should align the existing RN stuff to the new Android / iOS stuff.

IMO

Comment (tiagoengel) [reply decisions/components-preview.md]: I see what you mean but this is more about the previews than the actual component. If the actual component is different then by all means the preview should be different to. Maybe I should clarify that

Comment (georgegillams) [reply decisions/components-preview.md]: 👍

Comment (shaundon) [reply decisions/components-preview.md]: Yeah, doesn't hurt to make it more explicit 👍

File: decisions/components-preview.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Component Preview (a.k.a stories)
+
+## Decision
+- All previews should respond to a shortcut to hide the navbar (CMD+D).
+- The component being previewed should fit in the screen in portrait mode *without* the navbar.
+- The IOS preview should be as close as possible as the Android version (Always respecting any platform specific behaviour).
```
Comment (shaundon) [inline decisions/components-preview.md]: Nitpick: Can we make it `iOS` (lowercase `i`) 🙏 ?

File: decisions/components-preview.md (hunk 1 above)
Comment (shaundon) [inline decisions/components-preview.md]: Nitpick: `React Native` should be capitalised.

File: decisions/components-preview.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+- Diferent configurations (e.g. normal text and emphasized text) should have different previews. In this case clicking the main story should show a new list view with all configs.
```
Comment (shaundon) [inline decisions/components-preview.md]: * `Different` is misspelled.
* Nitpick: `emphasized` should be `emphasised` (British spelling).

**Comment by tiagoengel** [comment]
All addressed @georgegillams and @shaundon. I ended up removing the RN section as I might create confusion and not apply to most cases.

**Pull request merged**